src/main/java/io/buildpacks/example/sample/GreetingController.java

```diff
@@ -7,8 +7,14 @@
 @RestController
 public class GreetingController {
```

Comment:
2. It's a good practice to provide meaningful JavaDoc comments for methods, especially for public APIs like the `greeting` method.